plugins/spi/SPIDevice.h

```diff
@@ -86,7 +86,7 @@ class SPIDevice: public ola::Device {
   static const char SPI_DEVICE_NAME[];
   static const char HARDWARE_BACKEND[];
   static const char SOFTWARE_BACKEND[];
-  static const uint16_t MAX_GPIO_PIN = 25;
+  static const uint8_t MAX_GPIO_PIN = 1023;
```

Comment:
I had some git troubles, sorry. Will rectify in the morning.

On Wed, Sep 7, 2016 at 8:08 PM Peter Newman <notifications@github.com>
wrote:

> In plugins/spi/SPIDevice.h
> <https://github.com/OpenLightingProject/ola/pull/1097#discussion_r77932186>
> :
>
> > @@ -86,7 +86,7 @@ class SPIDevice: public ola::Device {
> >    static const char SPI_DEVICE_NAME[];
> >    static const char HARDWARE_BACKEND[];
> >    static const char SOFTWARE_BACKEND[];
> > -  static const uint16_t MAX_GPIO_PIN = 25;
> > +  static const uint8_t MAX_GPIO_PIN = 1023;
>
> This needs to be uint16_t. Again I'm not clear why Travis isn't catching
> it.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenLightingProject/ola/pull/1097/files/cc1b5b75e127c6c400f43556d309aa0b25f02f99..817cc70496dae8036d5ca0d4bea953d1e8ea6c16#r77932186>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEdWXlAPQLlaVjwsYHAcA75UOOlPVWXRks5qn2AJgaJpZM4JBreb>
> .
>